pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -892,6 +892,7 @@ spec:
                     values:
                     - {{ $targetAffinityVal }}
                 topologyKey: kubernetes.io/hostname
+                namespaces: [{{.Volume.runNamespace}}]
```

Comment:
What does this contain? Is it PVC namespace or is it openebs?

Reply:
PVC namespace. Without this change the controller pod will try to look for the application pod in openebs namespace.